app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -6,6 +6,10 @@ def call(toolbar_name)
     build_toolbar(toolbar_name)
   end
 
+  def call_by_class(toolbar_class)
+    build(toolbar_class)
+  end
```

Comment:
This one does not dynamically figure out which class to call, you're giving it the class that should be used to build the toolbar.